index.html

```diff
@@ -16,9 +16,61 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="white-font-style">
+    <div class="container">
+      <h2>Code Journal</h2>
```

Comment:
This doesn't line up with the content in the other container. You should put it in a `row` and an appropriately sized `column` class